css/index.css

```diff
@@ -1,3 +1,535 @@
+	margin-left: 0;
+}
+
+.sthird {
+	display: flex;
+	justify-content: space-between;
+}
+
+.sthird div, img {
+	width: 48%;
+}
+
+.services-info-img {
+	width: 500px;
+	height: 350px;
+}
+
+ul {
+	list-style-type: disc;
+}
+
+.athird {
+	width: 100%;
+	display: flex;
+	height: 1000px;
+	justify-content: space-between;
+	flex-flow: row wrap;
+}
+
+.athird img {
+	width: 500px;
+	height: 325px;
+}
+
+.container {
+	position: relative;
+	width: 50%;
+  }
+  
+  .image {
+	display: block;
+	width: 100%;
+	height: auto;
+  }
+  
+  .overlay {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	height: 98%;
+	width: 98%;
+	opacity: 0;
+	transition: .5s ease;
+	background-color: lightblue;
+  }
+  
+  .container:hover .overlay {
+	opacity: 1;
+  }
+  
+  .text {
+	color: white;
+	font-size: 20px;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	text-align: center;
+  }
+
+ #info{
+	 width: 100%;
+	 display: flex;
+	 justify-content: center
+	 
+ }
+
+ #info h2{
+	 width: 300px;
+	font-size: 40px;
+	background: lightblue;
+	border-radius: 10%;
+	display: flex;
+	justify-content: center;
+ }
+
+ .banner{
+	 width: 800px;
+ }
+
+
+ .hr{
+     display: flex;
+     justify-content: center;
+     justify-items: center;
+     margin: 0 auto;
+ }
+
+.hiddenLogo{
+visibility: hidden;
+height: 0;
+
+}
+ /* Responsive Design */
+
+ @media screen and (max-width: 800px){
+    body{
+        width: 800px;
+    }
+
+    .logo{
+        display: none;
+    }
+    
+    #navDiv{
+        width: 800px;
+        display: flex;
+        justify-content: space-between;
+        margin: 0px 20px;
+    }
+    nav{
+        width: 100%;
+        padding-right: 0%;
+        padding-left: 3%
+    }
+
+    nav a {
+        padding: 0;
+        
+    }
+
+    #circle{
+        display: none;
+    }
+
+    .first{
+        width: 800px;
+        display: flex;
+        justify-content: center;
+    }
+
+    #firsthalf{
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        justify-items: center;
+        margin: 0 auto;
+
+    }
+
+    #slogan{
+        margin-right: 0%;
+    }
+
+    #homeButton{
+        margin-left: 12%;
+    }
+
+
+
+	.second div{
+		width: 46%;
+	}
+
+	.second{
+		display: flex;
+		justify-content: space-between;
+		margin: 0 3%;
+		padding-right:0 9%;
+	}
+
+	.hiddenLogo{
+		display: flex;
+		justify-content: center;
+		align-content: center;
+		justify-items: center;
+		height: 100px;
+		visibility: visible;
+	}
+
+	.logoSmall{
+		width:175px;
+		height: 35px;
+		margin-left: 5%;
+		margin-top: 5%;
+		
+	}
+
+	header{
+		padding-top: 0;
+	}
+
+	.main{
+		margin-top: 0;
+	}
+
+	.middle-img{
+		padding-left: 1%;
+	}
+
+ }
+
+
+
+ @media screen and (max-width: 500px){
+	body{
+		width: 800px;
+	}
+
+	#navDiv{
+		width: 800px;
+		padding-right:0%;
+		padding-left: 0;
+		margin-left: 0;
+	}
+	nav{
+		display: flex;
+		flex-flow: row wrap;
+
+	}
+
+	nav a{
+		display: flex;
```

Comment:
Remember flexbox is a parent element property, so the use of it here doesn't really do anything because this is the inner most child of this component.